kickstart/PXELinux.erb

```diff
@@ -24,6 +24,8 @@ LABEL linux
     APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> ks.device=bootif network ks.sendmac
     <% elsif @host.operatingsystem.name != 'Fedora' and @host.operatingsystem.major.to_i >= 7 -%>
     APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> network ks.sendmac
+    <% elsif @host.operatingsystem.name =~ /.*Atomic.*/ -%>
```

Comment:
Also, I think the order of checks should be different -- I have both 'Fedora' and 'Atomic' in the os name, which results in Fedora kernel parameters being used.